blocks/v2-feature-carousel/v2-feature-carousel.js

```diff
@@ -0,0 +1,20 @@
+import { createElement, removeEmptyTags } from '../../scripts/common.js';
+
+const blockname = 'v2-feature-carousel';
+
+export default function decorate(block) {
+  const cardsWraper = block.firstElementChild;
+  cardsWraper.classList.add(`${blockname}__cards-list`);
+  cardsWraper.append(...cardsWraper.firstElementChild.children);
+
+  [...cardsWraper.querySelectorAll('p')].forEach((el) => {
```

Comment:
Which is the reason for using `p` instead of `ul > li`? Semantically speaking for me seems to be better having a list, but that's my personal opinion.

Reply:
Done